docs/source/sophronia.rst

```diff
@@ -0,0 +1,251 @@
+Sophronia
+=========
+
+*From ancient Greek: sensible, prudent.*
+
+
+This city processes each S2 signal previously selected as pmaps in
+irene assuming a unique S1 within an event to produce a set of
+reconstructed energy depositions (**Hits**). Hits consist of three
+dimensional coordinates with an associated energy (PMT signal) and
+charge (SiPM signal). The city contains a peak/event filter, which can
+be configured to find events with a certain number of S1/S2 signals
+that satisfy certain properties. Currently, the city is designed to
+accept only 1 S1 signal. If the configuration allows for more than one
+S1, the first one will be used to estimate z. Furthermore, hits can be
+optionally corrected for geometrical and lifetime effects. Besides
+hits, the city also stores the global (x, y) position of each S2
+signal.  The tasks performed are:
+    - Classify peaks according to the filter.
+    - Filter out events that do not satisfy the selector conditions.
+    - Resample S2 signals.
+    - Compute a set of hits for each slice in the resampled S2 signal.
+    - If there are more than one hit per slice, share the energy
+      according to the charge recorded in the tracking plane.
+    - If correction maps are provided, applies corrections to hits.
```

Comment:
For consistency, I would use `apply`, like the other verbs (or use the 3rd singular person for all of them). Alternatively, since you are describing the tasks, you could use names, such as classification, filtering... I personally prefer verbs, it is easier to read.